cpp/fetcher/fetcher.cc

```diff
@@ -74,9 +88,11 @@ struct FetchState {
 
 
 FetchState::FetchState(Database<LoggedCertificate>* db,
-                       unique_ptr<PeerGroup>&& peer_group, Task* task)
+                       unique_ptr<PeerGroup>&& peer_group,
```

Comment:
That reminds me... I've read about "how is a `unique_ptr` passed in" recently, and the consensus seems to be that you should prefer taking it by value (which is fine, because `unique_ptr` itself is move-constructible), because that guarantees that it's moved.

Taking by rvalue-ref is suggested for the cases where the ownership *might* be taken, depending on what code path is taken (doing `std::move(foo)` by itself does *nothing*, as you may remember), but this is generally thought of to be slightly weird (after the call, the pointer may or may not be valid), and to be avoided.